reviewer-2-comments.md

```diff
@@ -0,0 +1,13 @@
+# Reviewer 2
+## General comments
+I have reviewed your figure and I respect all the work that was put into it. 
+Incorporating a piece of fruit within a logo is certainly a novel approach.
+That said, I have some major concerns regarding the values that your peach conveys. 
+Thus, I have provided comments to help your peach better symbolize eternal youth and be more accessible for color-blind users.
+
+## Specific comments
+- Please change the color of the peach to [252,141,89]. Please refer to https://colorbrewer2.org for appropriate color combinations in the future.
```

Comment:
Does color = [252, 141, 89] actually run in matlab? I thought matlab does all their colors in fractions of 255 instead of the traditional RGB values. Adjust accordingly if it errors out, no need for extra reviewer 2 headaches. 😉

Reply:
this goes for leaf color as well

Reply:
I show that you have to divide the RGB values by 255 in the solution.

Reply:
sorry @ross-wilkinson, when i started the review, only one file was coming up as changed: `reviewer-2-comments.md`. I see that with commit 778815d, the code is changed as well. Ignore the RGB comments and I'll continue the review.